src/controllers/cron/update_local_smartapi.js

```diff
@@ -153,7 +153,19 @@ const getOpsFromPredicatesEndpoints = async (specs) => {
 
 const updateSmartAPISpecs = async () => {
     const SMARTAPI_URL = 'https://smart-api.info/api/query?q=tags.name:translator&size=1000&sort=_seq_no&raw=1&fields=paths,servers,tags,components.x-bte*,info,_meta';
-    const res = await axios.get(SMARTAPI_URL, { headers: { 'User-Agent': userAgent } });
+    const overridesPath = path.resolve(__dirname, "../config/smartapi_overrides.json");
```

Comment:
Currently syncing fails because this path is invalid. It should be `../../config/smartapi_overrides.json`. 

I'd recommend testing this PR using the `Sync SmartAPI` and `Sync SmartAPI (with overrides)` vscode debugging launchers, and intentionally breaking the request URL (taking out the ".info" should be sufficient) to test behavior on failed requests.